APIs/schemas/substream_constraints.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Describes substream constraints",
+  "title": "Constraint Set",
+  "type": "object",
+  "minProperties": 1,
+  "properties": {
+    "description": {
+      "description": "Substream group description",
+      "type": "string"
+    },
+    "format": {
+      "description": "Substream format",
+      "type": "string",
+      "enum": ["video", "audio", "data"]
```

I was thinking this:

```suggestion
      "enum": [
        "urn:x-nmos:format:video",
        "urn:x-nmos:format:audio",
        "urn:x-nmos:format:data"
      ]
```

Or possibly an "open enum", something like this:

```suggestion
      "anyOf": [
        {
          "enum": [
            "urn:x-nmos:format:video",
            "urn:x-nmos:format:audio",
            "urn:x-nmos:format:data"
          ]
        },
        {
          "pattern": "^urn:x-nmos:format:(?!mux$)"
        }
      ]
```